### ISSUE TITLE
Support for --testsuite phpunit parameter

### ISSUE THREAD

**Issue by chbiel** [brianium/paratest#142]
It would be nice to specify the --testsuite parameter and execute the tests in the testsuite in parallel.

E.g.
```xml
<testsuite name="MySuite">
    <file>MyFirstTest.php</file>
    <file>MySecondTest.php</file>
</testsuite>
```

I also have other suites so I only what to execute that one:
paratest --testsuite=MySuite --processes=2

Would be very nice :)

**Comment by chbiel** [comment]
added pull request to add this feature:
https://github.com/brianium/paratest/pull/144

**Comment by kick-the-bucket** [comment]
PHPUnit actually support filtering by multiple testsuites separated by comma which is currently not supported by paratest.
Maybe should this be reopened or a new issue created?
